txbits/app/wallet/Wallet.scala

```diff
@@ -304,6 +358,7 @@ class Wallet(rpc: JsonRpcHttpClient, currency: CryptoCurrency, nodeId: Int, para
   }
 
   private def getBalance = {
+    // This method is unused as balance is now tracked in the database
```

Comment:
So it would lead to an infinite loop of crashing and restarting? Can we handle that case by updating the balance in the db and going into low funds mode the same way it would behave if it ran out of money normally?